### PR TITLE
Add Czech translation

### PR DESCRIPTION
**Pull request by Tony763**

### CODE REVIEW

File: vocab/cs-cz/camera.intent [hunk 1]
```diff
@@ -0,0 +1,3 @@
+show (me|) (the|a|an|) (latest|actual|) (picture|still|shot|snapshot) of {Entity}.
```
Comment (pfefferle) [inline vocab/cs-cz/camera.intent]: I think this is wrong?!?

**Comment by pfefferle** [comment]
@Tony763 you opened the PR to the master instead of the camera branch ☺️

**Comment by Tony763** [comment]
ou, will fix :D

File: vocab/cs-cz/camera.intent (hunk 1 above)
Comment (Tony763) [reply vocab/cs-cz/camera.intent]: yep, fixed

**Pull request closed without merging**